queries/document.py

```diff
@@ -138,7 +138,7 @@ def construct_insert_document(document_name,
 
 def construct_attach_document_to_unsigned_version(doc_uri, prev_ver_doc_uri, graph=APPLICATION_GRAPH):
     """Also handles attaching the signed version to the previous one's case
-    The relation ?case dossier:Dossier.bestaatUit $doc is what lets
+    The relation $doc sign:ongetekendStuk $prev_doc is what lets
```

Comment:
Looking at this now. I didn't really verify Yggdrasil would be able to pick up.
But it should work fine since any signedPieces coming from our `pdf-signature-remover`  are being propagated.
In yggdrasil, look for `collectReleasedDocuments`.
Setting this relation in this service was added 5 years ago. The code in yggdrasil was updated 2 years ago.